### PR TITLE
Link component

### PR DESCRIPTION
**Pull request by leon-up9**

### CODE REVIEW

File: ui-common/src/components/UI/index.ts [hunk 1]
```diff
@@ -5,10 +5,10 @@ import Tooltip from "./Tooltip";
...
 import { StatusBar } from "./StatusBar";
 import CustomModal from "./CustomModal";
-import { InformationIcon } from "./InformationIcon";
+import { InformationIcon, Link } from "./InformationIcon";
 import SelectList from "./SelectList";
 import NoDataMessage from "./NoDataMessage";
 
 
-export { LoadingOverlay, Select, Tabs, Tooltip, Checkbox, CustomModal, InformationIcon, SelectList, NoDataMessage }
+export { LoadingOverlay, Select, Tabs, Tooltip, Checkbox, CustomModal, InformationIcon, SelectList, NoDataMessage, Link }
```
Comment (IgorGov) [inline ui-common/src/components/UI/index.ts:13]: can 'StatusBar' be part of this export?

File: ui-common/src/components/UI/InformationIcon.tsx [hunk 2]
```diff
@@ -3,17 +3,24 @@ import styles from "./style/InformationIcon.module.sass"
...
-            <span>Docs</span>
+        <a href={link} className={className} title={title} target="_blank">
+            {children}
         </a>
     </React.Fragment>
 }
 
-
+export const InformationIcon: React.FC<LinkProps> = ({ className }) => {
+    return <React.Fragment>
```
Comment (IgorGov) [inline ui-common/src/components/UI/InformationIcon.tsx:21]: why do we need to use fragment here?
according to the documentation it is a tool for specific scenario which is not related in this case
https://reactjs.org/docs/fragments.html

File: ui-common/src/components/UI/InformationIcon.tsx [hunk 3]
```diff
@@ -3,17 +3,24 @@ import styles from "./style/InformationIcon.module.sass"
...
+        <Link title="documentation" className={`${styles.linkStyle} ${className}`} link={DEFUALT_LINK}>
+            <span>Docs</span>
+        </Link>
+    </React.Fragment>
+}
```
Comment (IgorGov) [inline ui-common/src/components/UI/InformationIcon.tsx:26]: new line

File: ui-common/src/components/UI/index.ts (hunk 1 above)
Comment (leon-up9) [reply ui-common/src/components/UI/index.ts:13]: yes technically,
its was a aesthetic separation for a easily identify UI components and others

**Review by lirazyehezkel — APPROVED** [review]

**Pull request merged**